keps/sig-cloud-provider/20191004-out-of-tree-credential-providers.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Out-of-Tree Credential Providers
+authors:
+  - "@mcrute"
+  - "@nckturner"
+owning-sig: sig-cloud-provider
+participating-sigs:
+  - sig-node
+  - sig-auth
+reviewers:
+  - "@andrewsykim"
+  - "@cheftako"
+  - "@nckturner"
```

Comment:
You can add me as a reviewer. And you probably shouldn't be an author and reviewer :wink: